cloudflare-mixin/dashboards/cloudflare-worker-overview.libsonnet

```diff
@@ -0,0 +1,399 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
```

Comment:
Not sure why, but when entering the page, the job / isntance presets aren't automatically selected. This might just be an issue on my computer though?

Reply:
I'll update the defaults

Reply:
no clue, lets see what the reviewers say in this case.